doc/tutorials/content/sources/qt_colorize_cloud/CMakeLists.txt

```diff
@@ -1,28 +1,32 @@
-cmake_minimum_required (VERSION 2.6 FATAL_ERROR)
+cmake_minimum_required(VERSION 2.8.11)
+
+project(pcl_colorize_cloud)
+
+# init_qt: Let's do the CMake job for us
+set(CMAKE_AUTOMOC ON) # For meta object compiler
+set(CMAKE_AUTORCC ON) # Resource files
+set(CMAKE_AUTOUIC ON) # UI files
+
+# Find includes in corresponding build directories
+set(CMAKE_INCLUDE_CURRENT_DIR ON)
+
+# Find the QtWidgets library
+find_package (Qt5 REQUIRED Widgets)
 
-project      (pcl-colorize_cloud)
-find_package (Qt4 REQUIRED)
 find_package (VTK REQUIRED)
 find_package (PCL 1.7.1 REQUIRED)
 
-include_directories (${PCL_INCLUDE_DIRS})
-link_directories    (${PCL_LIBRARY_DIRS})
-add_definitions     (${PCL_DEFINITIONS})
+# Fix a compilation bug under ubuntu 16.04 (Xenial)
+list(REMOVE_ITEM PCL_LIBRARIES "vtkproj4")
 
-set  (project_SOURCES main.cpp pclviewer.cpp)
-set  (project_HEADERS pclviewer.h)
-set  (project_FORMS   pclviewer.ui)
-set  (VTK_LIBRARIES   vtkRendering vtkGraphics vtkHybrid QVTK)
-
-QT4_WRAP_CPP (project_HEADERS_MOC   ${project_HEADERS})
-QT4_WRAP_UI  (project_FORMS_HEADERS ${project_FORMS})
+include_directories(${PCL_INCLUDE_DIRS})
+link_directories(${PCL_LIBRARY_DIRS})
+add_definitions(${PCL_DEFINITIONS})
 
-INCLUDE         (${QT_USE_FILE})
-ADD_DEFINITIONS (${QT_DEFINITIONS})
+set  (project_SOURCES main.cpp pclviewer.cpp)
 
-ADD_EXECUTABLE  (colorize_cloud ${project_SOURCES}
-                                ${project_FORMS_HEADERS}
-                                ${project_HEADERS_MOC})
+ADD_EXECUTABLE  (pcl_colorize_cloud ${project_SOURCES})
 
-TARGET_LINK_LIBRARIES (colorize_cloud ${QT_LIBRARIES} ${PCL_LIBRARIES} ${VTK_LIBRARIES})
+TARGET_LINK_LIBRARIES (${PROJECT_NAME} ${QT_LIBRARIES} ${PCL_LIBRARIES})
```

Comment:
I believe explicitly linking against `QT_LIBRARIES` is not needed, the `qt5_use_modules` command below should take care of this.

Also, since you are rewriting most of the content anyway, can you please convert CMake function calls to lowercase? Uppercasing is an [outdated practice](https://cmake.org/gitweb?p=cmake.git;a=commitdiff;h=77543bd).

Reply:
Ok. Should I also take care of spaces before parenthesis?